### DISCUSSION TITLE
Spherical model

### DISCUSSION THREAD

**Discussion by VictorS-M** [simnibs/simnibs#479] [2. General]
Hello to everyone!!!
I'm trying to do a TMS  simulation over a homogeneus spherical model. But the problem is that I can not run the simulation only with one layer and when I make the simulation with two but with the same conductivity the electric field behaves like there is a interface with different conductivities.
Can anyone tell me if it is possible to model a homogeneus sphere?
Thank you so much.

**Comment by bjornsig** [comment]
Hello Victor,

The reason is it wants to have a skin and the gray matter however setting both to the same conductivity should work. Can you share the sphere and simulation script and a screenshot of where the field behaves strangely?

**Reply by VictorS-M** [reply]
Thank you very much for your reply.
I have set the same conductivity and I know that everything works fine. The change in the electric field was due to a small change in conductivity. 
Thanks for your help.
